### PR TITLE
Self hoster data optin

### PR DESCRIPTION
**Pull request by jparyani**
This will send a notification to admins of a Sandstorm server, asking if they wish to opt-in to stats collection.

### CODE REVIEW

File: shell/client/shell.html [hunk 1]
```diff
@@ -950,6 +956,7 @@ <h4 class="share-with-others">Share with others</h4>
 
 <template name="adminStats">
   <div id="stats">
+    <label><input class="enableStatsCollection" type="checkbox" name="reportStats" checked={{reportStats}}>Report anonymized stats below to sandstorm.io</label>
```
Comment (kentonv) [inline shell/client/shell.html]: I'd like to present a yes/no decision here, where they must click on one for the notification to disappear.

I just sent an e-mail to the mailing list to propose text and see what people think.

File: shell/shared/stats.js [hunk 2]
```diff
@@ -137,6 +137,23 @@ if (Meteor.isServer) {
     record.computeTime = Date.now() - now;
 
     ActivityStats.insert(record);
+    if (ActivityStats.find().count() > 3) {
+      var reportSetting = Settings.findOne({_id: "reportStats"});
+      if (!reportSetting) {
+        // Setting not set yet, send out notifications and set it to false
+        globalDb.sendAdminNotification("Sandstorm.io would like you to opt-in to stats " +
```
Comment (kentonv) [inline shell/shared/stats.js]: This sounds a bit scary. How about: "You can help Sandstorm by sending us some anonymous usage stats. Click here for more info."

**Comment by ocdtrekkie** [comment]
I'm fundamentally happy with this in any language as long as it's not on by default. :)

**Comment by jparyani** [comment]
Ready to merge. I changed around the text a bit on the stats page as well, but feel free to make your own edits.

**Pull request merged**